cmd/clusterctl/cmd/config_cluster.go

```diff
@@ -19,6 +19,10 @@ package cmd
 import (
 	"fmt"
 	"os"
+	"sort"
```

Comment:
/hold

this needs to be moved to generate now that the config command is deprecated (see https://github.com/kubernetes-sigs/cluster-api/pull/4584/files#diff-adebc79dc951a4b0b094a3e1462bce8f2bb090a9dcdffcdcbc4b69e13d7bb7f3R29) - sorry for the added work there, the PR merged before this one had a chance to go in

Reply:
No problem. I moved that code from `clusterctl config` to `clusterctl generate` and rebased.